homework21/tests/test_deposit.py

```diff
@@ -0,0 +1,112 @@
+"""Homework 21"""
+# Покройте тестами програмы: "Библиотека" и "Банковский вклад" из домашней
+# работы №11:
+
+# Создайте отдельный модуль для тестов.
+# Импортируйте функции из модуля.
+# Напишите тесты на проверку работоспособности приложения.
+# Тесты должны проверять как положительный результат, так и отрицательный.
+# Тесты используют библиотеку logging\loguru для логирования
+# Добавьте простой отчет к вашим тестам используя библиотеку pytest-html
+# https://pytest-html.readthedocs.io/en/latest/
+# https://pytest-with-eric.com/plugins/pytest-html/
+# Тесты запускаются с помощью pytest
+# Добавьте в файл .gitignore все файлы и папки которые генерирует в процессе
+# своей работы pytest и pytest-html, те в репозитории с кодом не должно быть
+# лишних файлов
+# Есть возможность запустить тесты с разным уровнем логгирования без изменения
+# кода(те передача уровня логгирования во время запуска тестов)
+# Примерная структура проекта:
+# homework21/
+# .github/
+# - ...
+# - tests/
+# -- test_<bank-app>.py    <-- Файл с тестами для тестирования банка
+# -- test_<library-app>.py <-- Файл с тестами для тестирования библиотеки
+# - source/
+# -- <library>.py          <-- Модуль реализуюший функционал библиотеки
+# -- <bank>.py             <-- Модуль реализуюший функционал банка
+# .gitignore
+# requirements.txt
+# ...
+
+# Note: Помимо кода, пулл реквест должен содержать в себе файл-репорт
+# (прикрепленный к пулл реквесту) который генерируется по результатам запуска
+# тестов.
+
+# Note2: Названия файлов примерное
+
+
+import pytest
+from logs.logger import setup_logger
+
+from homework21.source.deposit import Bank
+
+logger = setup_logger('Deposit tests', 'logs/my_log.log')
```

Comment:
Just pass module name, and tests do not need to pass path to log file

```suggestion
logger = setup_logger(__name__)
```